src/components/form/FormFieldVoucher.vue

```diff
@@ -49,9 +51,15 @@ export default defineComponent({
   props: {},
   setup() {
     const formFieldTextRequiredRef = ref(null);
-    const defaultPaymentAmountMin = parseInt(
-      rideToWorkByBikeConfig.entryFeePaymentMin,
-    );
+    const challengeStore = useChallengeStore();
+    const defaultPaymentAmountMin = computed(() => {
+      const currentPriceLevels = challengeStore.getCurrentPriceLevels;
```

Comment:
This code lines 56-62 should be stand alone function, because it is repeated across components code, see `RegisterChallengePayment` component code line 89.

Reply:
Fixed with commit https://github.com/auto-mat/ride-to-work-by-bike-frontend/pull/814/commits/7671dbde12c708b595cf8e9ceff791533fa5bd0e.